src/sentry/static/sentry/app/views/groupEvents.jsx

```diff
@@ -82,9 +82,11 @@ const GroupEvents = React.createClass({
           pageLinks: jqXHR.getResponseHeader('Link')
         });
       },
-      error: (error) => {
+      error: (err) => {
+        let error = err.responseJSON || true;
+        error = error.detail || true;
```

Comment:
Nah, I just assume JavaScript similar to literally any other programming language that would yell trying to access a property of a boolean. :)